### DISCUSSION TITLE
Option to play music from a custom source

### DISCUSSION THREAD

**Discussion by Signupking5000** [jagrosh/MusicBot#1747] [Ideas]
It would be great if there was the option to let the bot play songs from a privat server without having to enter the full file in the discord chat. Just like it now searches on Youtube but instead on my own server.

**Comment by sad-cactus** [comment]
What I did was create a playlist file with the file names, not quite what your looking for but it works similarly.

**Reply by sad-cactus** [reply]
locally of course.
